Packs/FeedNVDv2/Integrations/FeedNVDv2/FeedNVDv2.py

```diff
@@ -0,0 +1,421 @@
+    through the date the fetch-indicators or nvd-get-indicators command is 
+    called
+
+    Args:
+        client: An instance of the BaseClient connection class
+
+    Returns:
+        Total number of CVE indicators fetched
+
+    """
+    url = "/rest/json/cves/2.0/?noRejected"
+    param = {}
+    start_index = 0
+    results_per_page = 2000
+    raw_cves = []  # type: ignore
+
+    start_date = datetime.fromisoformat(str(start_date))
+    end_date = datetime.fromisoformat(str(end_date))
+
+    param['lastModStartDate'] = start_date.strftime(DATE_FORMAT)  # type: ignore
+    param['lastModEndDate'] = end_date.strftime(DATE_FORMAT)  # type: ignore
+
+    if client.has_kev:
+        url += '&hasKev'
+
+    param['startIndex'] = int(start_index)  # type: ignore
+    param['resultsPerPage'] = results_per_page  # type: ignore
+
+    more_to_process = True
+
+    # Collect all the indicators together
+    while more_to_process:
+        try:
+            if not test_run:
+                res = client.http_get(url, param)
+            else:
+                with open('./Packs/FeedNVDv2/Integrations/FeedNVDv2/test_data/test_cve_data.json', encoding='utf-8') as f:
+                    res = json.loads(f.read())
+                    more_to_process = False
+            if "error" in res:
+                return_error(res.get('error'))
+            total_results = res.get('totalResults', 0)
+
+            if total_results:
+                raw_cves += res.get('vulnerabilities')
+
+                param['startIndex'] = str(int(param['startIndex']) + results_per_page)
+
+            if (int(param['startIndex']) >= total_results):
+                more_to_process = False
+
+        except Exception as e:  # pylint: disable=broad-except
+            demisto.debug(e)
+        finally:
+            time.sleep(.5)
+
+    return raw_cves
+
+
+def fetch_indicators_command(client: Client, test_run: bool):
+    """
+    Initiates the CVE retrieval process while tracking the total run time and indicator fetch count
+
+    Args:
+        client: An instance of the BaseClient connection class
+
+    Returns:
+        Total number of CVE indicators fetched
+
+    """
+
+    parsed_cves: list = []
+    temp_cves: list = []
+    exceeds_span = True
+    iteration = 0
+    command = demisto.command()
+
+    s_date = client.start_date
+    try:
+        date.fromisoformat(s_date)
+    except ValueError:
+        return_error("Incorrect date format specified. Should be in the format of YYYY-MM-DD")
+    start_date = datetime.strptime(s_date, "%Y-%m-%d")
+
+    last_run_data = demisto.getLastRun()
+
+    if last_run_data:
+        last_run = parse(last_run_data.get("lastRun"))
+
+    if command == 'nvd-get-indicators':
+        history = demisto.getArg('history')
+        demisto.debug(f'Retrieving last {history} days of CVEs using nvd-get-indicators')
+        last_run = (datetime.now() - timedelta(days=int(history)))
```

Comment:
Won't running this command as it is now will change the feed last run? Is that on purpose? It will affect the feed fetch which might seem like a bug to the client.

Reply:
No, last_run is simply a variable name used here and I am not calling the demisto.updateLastRun method

Reply:
So please rename it as it may confuse